src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar">
+      <div
+        class="day calendar__day1 day--mon day--tue day--wed day--thu day--fri day--sat day--sun"
```

Comment:
The class names 'day--mon', 'day--tue', 'day--wed', 'day--thu', 'day--fri', 'day--sat', and 'day--sun' are all applied to a single div. This is likely incorrect as each day should have its own div with the appropriate class name. Consider separating these into individual div elements for each day of the week.